submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,272 @@
+- **Cost of processing setup:** This is in terms of materials and space needed to make enough buildings and logistics for the processing, as well as the unlock cost of the required technology. Each tier of processing adds to these costs.
+- **Availability of more coal/oil to just use instead:** Since these resources are technically infinite on most maps, you may prefer to skip the energy gains and brute force your fuel shortages by setting up more resource extraction infrastructure instead of more fuel processing infrastructure. This factor simply depends on your surroundings.
+- **How long you will keep using the burner devices:** If you are going to switch to electrical machines and alternate power sources soon, it may not be worth the effort of setting up a large fuel processing system.
+
+Since the above factors depend on your map and your plans, the conclusion varies on whether fuel processing is useful. Based on my experiences, I can offer the following advice about processing fuels:
+
+- **Processed fuels are excellent for trains and tanks.** The speed gains from rocket fuel is quite satisfying and nuclear fuel is on another level…
+- **You already need to mass produce solid fuel and rocket fuel to make rocket parts,** so it is not too difficult to produce a little more for your vehicle or other burner device needs.
+- **If you are burning coal for power, processing it into solid fuel will cut the coal demand by half.** And of course, you can go all out and run your boilers with nuclear fuel to reduce the demand to about 1/20th. However, unlocking coal liquification or nuclear fuel requires production science, which means you can unlock solar and (regular) nuclear power much sooner and [their advantages](https://www.reddit.com/r/technicalfactorio/comments/srosza/coal_vs_solar_vs_nuclear_setup_costs_and_running/) may make it worthwhile to switch to them before upgrading your coal setup. Nuclear power is [easier to do than it may seem]([Alt-F4 #57 - Nuclear Energy and You](https://alt-f4.blog/ALTF4-57/#)), as it provides massive energy yields for very few reactors and centrifuges, without needing Kovarex Enrichment!
+- **Use modules in your fuel processing to increase energy gains.** As previously touched upon, filling all the fuel processing machines with efficiency 1 modules is quick and offers some energy savings. More complex module combinations may offer better gains, as presented in [this guide]([Tutorial:Producing power from oil - Factorio Wiki](https://wiki.factorio.com/Tutorial:Producing_power_from_oil)) on the official wiki.
+
+## Summary
+
+This article started with a little burning question that brought more burning questions and evolved into an exploration of all of Factorio's fuels. We uncovered the following:
+
+**(1) Coal vs. coal liquification:** Liquifying coal and directly converting it into solid fuel yields 1.57 times as much energy per coal.
```

Comment:
```suggestion
**(1) Coal vs. coal liquification:** Liquifying coal and directly converting it into solid fuel yields 1.57 times as much energy per coal.
```
You previously said

> So we see that simply liquifying the coal gets us about 57% more energy out of it.

So I'd either turn this "1.57 times as much energy" into "57% more energy" or the other way around for consistency.